packages/shared/src/mdx/components/codeBlock.js

```diff
@@ -30,12 +30,35 @@ const CodeBlock = ({ children: exampleCode, ...props }) => {
   if (props['react-live']) {
     return <LoadableComponent code={exampleCode} />;
   } else {
+    const coloredIntervals = {};
+
+    if (props.coloredLines) {
+      const iterations = props.coloredLines.split(",");
+      iterations.forEach(iteration => {
+        const parts = iteration.split("|");
```

Comment:
```
const [range, color] = iteration.split("|")
if (!color) {
  return //just in case some makes a mistake
}
 
coloredIntervals[range] = color;
```